recipes/pystreamer/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "pyStreamer" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e9ff94c97517a686d41e58fa163e543a340bf802b9c5a911aa7c4833c05c29e4
+  
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - python >=2.7
```

Comment:
the ``python >=2.7`` is not necessary, please remove it

Reply:
I'd like to restrict my package to python 2.7 and python 3.x only. No versions <=2.6

Reply:
Conda-forge only supports 2.7, 3.6 and 3.7, this selector is already the global default and thus redundant 😉